### PR TITLE
Properly assign cards to the new supporter when merging supporters

### PR DESCRIPTION
**Pull request by clarissalimab**
Addresses the problem in [tix #2682](https://github.com/CommitChange/tix/issues/2682).

### CODE REVIEW

File: spec/lib/merge_supporters_spec.rb [hunk 1]
```diff
@@ -139,6 +140,16 @@
...
+
+    it 'updates the card information on the supporter' do
+      MergeSupporters.update_associations([old_supporter1.id, old_supporter2.id], new_supporter.id, np.id, profile.id)
+      expect(new_supporter.reload.cards.first.id).to eq(card.reload.id)
+    end
+
+    it 'updates the supporter information on the card' do
+      MergeSupporters.update_associations([old_supporter1.id, old_supporter2.id], new_supporter.id, np.id, profile.id)
+      expect(card.reload.holder_id).to eq(new_supporter.reload.id)
+    end
```
Comment (clarissalimab) [inline spec/lib/merge_supporters_spec.rb]: I'm not sure if both tests are necessary, but I did them just to be safe.

**Comment by wwahammy** [comment]
Is this ready for my review?

**Review by wwahammy — APPROVED** [review]
I did additional changes!

**Comment by wwahammy** [comment]
@clarissalimab since I can't actually request your review,I was hoping you could review the changes I made to your changes. It would be helpful to know what you think and see if they make sense.

**Comment by clarissalimab** [comment]
@wwahammy I think they look great!

**Pull request merged**